src/test/java/com/rakovets/course/java/core/practice/date_and_time/DateWrapperTest.java

```diff
@@ -0,0 +1,95 @@
+package com.rakovets.course.java.core.practice.date_and_time;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.LocalDate;
+import java.time.temporal.Temporal;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.stream.Stream;
+
+public class DateWrapperTest {
+    @Test
+    void getDateTest() {
+        LocalDate localDate1 = DateWrapper.getLocalDate(1994, 7, 12);
+        assertEquals("1994-07-12", localDate1.toString());
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"